cogs5e/utils/gameutils.py

```diff
@@ -16,3 +32,84 @@ async def send_hp_result(ctx, caster, delta=None):
         await caster.message_controller(ctx, f"{caster.name}'s HP: {caster.hp_str(True)}{deltaend}")
     else:
         await ctx.send(f"{caster.name}: {caster.hp_str()}{deltaend}")
+
+
+async def send_current_coin(ctx, character, coin="All"):
+    """
+    Sends the current contents of the CoinPurse
+    """
```

Comment:
```suggestion
async def send_current_coin(ctx, character, coin=None):
    """
    Sends the current contents of the CoinPurse.
    If ``coin`` is passed, it must be a valid coin type and will only show the amount of that specific coin.
    """
    if coin is not None and coin.lower() not in CoinTypes:
        raise ValueError(f"{coin!r} is not a valid coin type.")
```
We should have a little bit more validation here, and `None` as a default value is a little bit cleaner than `"All"`